_maps/map_files/BoxStation/BoxStation.dmm

```diff
@@ -34158,7 +34158,9 @@
 /obj/structure/sign/warning/vacuum/external{
 	pixel_y = 32
 	},
-/turf/open/floor/engine/vacuum,
+/turf/open/floor/engine/vacuum{
+	thermal_conductivity = 0
+	},
```

Comment:
Could you make this a subtype instead of a var-edit?

Reply:
yeah the idea was to change the walls and floors used in the atmos and toxins incinerator with custom ones that would be subtypes of the base wall and floor. But that'll come later if the TM for performances goes well

Reply:
It shouldn't affect performance at all.

Reply:
if you mean this change in particular, i know it wont. i was meaning to the whole PR

Reply:
Pending.

Reply:
yep the change to the other maps will be done if the metastation SM changes will be approved, since on that map i removed the snowflake code and added projectors to toxins and atmos to be used inside the chamber to protect the users from the heat

Reply:
This could be resolved, i added a machine to all the maps incinerators and SM